project/threads/factory.py

```diff
@@ -0,0 +1,19 @@
+import factory
+from . import models
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = models.User
```

Comment:
What model in the threads app are we building this factory for? See `project/threads/models.py` where the models are defined.